レイアウトの作成

nukomaru0930 opened the pull request:
<!-- GitHub Copilot コードレビューへの指示: このプルリクエストをレビューしてコメントするときは日本語でお願いします。 -->

feature/#112

<!-- 関連するIssueやチケットのリンクを貼る。close #イシュー番号でPRが閉じたタイミングでイシューを閉じれる -->

### イシューリンク

### イシュー番号(自動クローズ用)

close #112

## やったこと

<!-- このPRで何をしたのか？ -->

ログイン後共通画面のレイアウトの作成

## やらないこと

<!-- このPRでやらないことは何か？ -->

無し

## できるようになること（ユーザ目線）

ヘッダーとメニューバーのレイアウト化

## できなくなること（ユーザ目線）

無し

## 動作確認

レイアウト化する前の画面との動作比較

## 影響範囲

<!-- 影響を及ぼす範囲や他の機能への影響 -->

タイムラインと共通のUIしている画面すべて

## テスト

<!-- テスト方法や結果 -->

なし

## 備考

<!-- レビュワーへの伝達事項や残しておきたい情報 -->

- レビュワーへの参考情報（実装上の懸念点や注意点などあれば記載）

nukomaru0930 committed:
feat:タイムライン関係のレイアウトの作成と修正feature/#112

nukomaru0930 committed:
refactor:レビュー内容の修正 feature/#112

hmuto111 commented on frontend/src/components/layouts/appLayout/appLayout.tsx:
```diff
@@ -0,0 +1,21 @@
+import { Header } from "@/components/ui/header/header";
+import { Menu } from "@/features/timeline/components/menu/menu";
+import styles from "./appLayout.module.css";
+
+type Props = {
+  children: React.ReactNode;
+};
+
```
これはみなくてもいいです

hmuto111 commented on frontend/src/components/layouts/appLayout/appLayout.tsx:
```diff
@@ -0,0 +1,21 @@
+import { Header } from "@/components/ui/header/header";
+import { Menu } from "@/features/timeline/components/menu/menu";
+import styles from "./appLayout.module.css";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+export const AppLayout = ({ children }: Props) => {
+  return (
+    <div className={styles.appLayoutContainer}>
+      <Header />
+      <div className={styles.appLayoutArea}>
+        <div className={styles.appLayoutLeft}>
+          <Menu />
+        </div>
+        <div className={styles.appLayoutRight}>{children}</div>
+      </div>
+    </div>
+  );
+};
```
ドキュメント化はいったんしなくていいです。UI実装のみで

hmuto111 reviewed (commented):
特に修正はないですが、timeLineAreaLeftなど意図の伝わりづらいものは、良かったらクラスネームを考え直してもらえると助かる

nukomaru0930 committed:
refactor:cssのクラスネーム修正しました feature/#112

nukomaru0930 committed:
refactor:cssのクラスネーム付け忘れてたので修正しました feature/#112

hmuto111 reviewed (approved):
LGTM

nukomaru0930 committed:
refactor: timeline.module.css→styleの名前がわかりずらかったので直しました。timeline.tsxもそれに合わせて修正は言ってます。Container内のheightいらなかったので消しました。
menu.tsx→フォルダの位置をUIに移動しました。その関係でcssも移動してるのと使ってるコードは更新かかってます。
appLayout.module.css→overflowがautoだったのでhiddenに変更しました。
feature/#112

nukomaru0930 committed:
refactor:cssのクラス名を最新版保存し忘れてコミットしてしまっていたため追加のコミットになります
feature/#112

nukomaru0930 committed:
refactor: timeline.module.cssのコンフリ起こしてた箇所の修正 feature/#112
